config/application.rb

```diff
@@ -5,10 +5,10 @@
 require 'rack-proxy'
 
 class Proxy < Rack::Proxy
-   def perform_request(env)
-     request = Rack::Request.new(env)
-     @app.call(env) unless request.path =~ %r{^/api}
-   end
+  def perform_request(env)
+    request = Rack::Request.new(env)
+    @app.call(env) unless request.path =~ %r{^api}
```

Comment:
I have a feeling that this might not work, since the 'request.path' wont have 'api' in it if we are going with http://api.redeemify.com/users/1.